cmd/util/cmd/epochs/cmd/recover.go

```diff
@@ -71,7 +71,9 @@ func addGenerateRecoverEpochTxArgsCmdFlags() error {
 	generateRecoverEpochTxArgsCmd.Flags().BoolVar(&flagAnInsecure, "insecure", false, "set to true if the protocol snapshot should be retrieved from the insecure AN endpoint")
 	generateRecoverEpochTxArgsCmd.Flags().IntVar(&flagCollectionClusters, "collection-clusters", 0,
 		"number of collection clusters")
-	// required parameters for network configuration and generation of root node identities
+	// Required parameters for network configuration and generation of root node identities
+	// It is expected that these flags may point to directory which includes extra node identities
```

Comment:
```suggestion
	// Required parameters for network configuration and generation of root node identities.
	// It is expected that these flags may point to directory which includes extra node identities,
```
Instead of a code comment, we could include this as valuable context in the `usage` string